ggml/src/ggml-vulkan/CMakeLists.txt

```diff
@@ -48,6 +50,7 @@ if (Vulkan_FOUND)
         message(STATUS "GL_NV_cooperative_matrix2 not supported by glslc")
```

Comment:
Should there be an "OFF" here like there is for coopmat1 above?

Reply:
Good catch!
I got too fast on it, thanks for catching it!

It would not be consistent otherwise => won't cause issues as default will be OFF anyway, but I think it is better to have it explicit.